DigitalOffice/Tests/ProjectServiceTests/Commands/DisableWorkersInProjectCommandTests.cs

```diff
@@ -0,0 +1,85 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.ProjectService.Commands;
+using LT.DigitalOffice.ProjectService.Commands.Interfaces;
+using LT.DigitalOffice.ProjectService.Models;
+using LT.DigitalOffice.ProjectService.Repositories.Interfaces;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.ProjectServiceUnitTests.Commands
+{
+    class DisableWorkersInProjectCommandTests
+    {
+        private WorkersIdsInProjectRequest workersProjectRequest;
+        private IDisableWorkersInProjectCommand command;
+        private Mock<IProjectRepository> repositoryMock;
+        private Mock<IValidator<WorkersIdsInProjectRequest>> validatorMock;
+
+        [SetUp]
+        public void SetUp()
+        {
+            var projectId = Guid.NewGuid();
+
+            repositoryMock = new Mock<IProjectRepository>();
+            validatorMock = new Mock<IValidator<WorkersIdsInProjectRequest>>();
+
+            command = new DisableWorkersInProjectCommand(repositoryMock.Object, validatorMock.Object);
+
+            var workerIds = new List<Guid>
+            {
+                Guid.NewGuid(),
+                Guid.NewGuid(),
+                Guid.NewGuid()
+            };
+
+            workersProjectRequest = new WorkersIdsInProjectRequest
+            {
+                ProjectId = Guid.NewGuid(),
+                WorkersIds = workerIds
+            };
+        }
+
+        [Test]
+        public void ShouldThrowValidationExceptionWhenRequestIsNotValid()
+        {
+            validatorMock
+                .Setup(x => x.Validate(It.IsAny<IValidationContext>()).IsValid)
+                .Returns(false);
```

Comment:
This should be addressed at the call.
Newest version of the library disabled this kind of mock and I think we should use up-to-date packages and test them appropriately. For the 9.0.1. version that mock works, of course, yeah; But when we update to 9.1+ there **will** be problems.